lib/secrets/secret-credentials.ts

```diff
@@ -0,0 +1,87 @@
+import { Stack, Tags } from 'aws-cdk-lib';
+import { ISecret, Secret } from 'aws-cdk-lib/aws-secretsmanager';
+import { OpenIdConnectProvider } from 'aws-cdk-lib/aws-iam';
+import { GitHubActionsFederateIntegrationForBranchesAndTags } from './gha-federate-access';
+
+export class AWSSecretsJenkinsCredentials {
+    static centralPortalMavenUsername: Secret;
+
+    static centralPortalMavenPassword: Secret;
+
+    static snapshotsMavenUsername: ISecret;
+
+    static snapshotsMavenPassword: ISecret;
+
+    constructor(stack: Stack) {
+      AWSSecretsJenkinsCredentials.snapshotsMavenPassword = Secret.fromSecretNameV2(stack, 'imported-maven-snapshots-password', 'maven-snapshots-password');
+
+      AWSSecretsJenkinsCredentials.snapshotsMavenUsername = Secret.fromSecretNameV2(stack, 'imported-maven-snapshots-username', 'maven-snapshots-username');
+
+      AWSSecretsJenkinsCredentials.centralPortalMavenUsername = new Secret(stack, 'maven-central-portal-username', {
```

Comment:
Since we create new secrets this and below, will they be override every deployment? For above two secrets, seems like we are importing from existing.